kinto_emailer/tests/test_includeme.py

```diff
@@ -348,6 +348,16 @@ def test_send_notification_calls_the_mailer_queue_if_configured(self):
             assert get_mailer().send_to_queue.called
 
 
+class ContextContentTest(unittest.TestCase):
+    def test_context_contains_settings(self):
```

Comment:
Looks like this is in effect a unit test of `context_from_event`. Would it be worth adding an integration test so that we call `send_notification` with a template that pulls from settings, the make assertions about the email? For example if the subject included `"subject": "Email from {settings.project_name}"`, assert that `foo_email['subject'] == "Kinto DEV"`